modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,289 @@
+|
+
+|===
+====
+
+
+[role=label--enterprise]
+=== Removed Neo4j metrics
+
+.See all removed metrics in Neo4j 2025.01
+[%collapsible]
+====
+[options="header"]
+|===
+|Name
+
+a|**link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#raft-core-metrics[Raft core metrics] - replaced accordingly by the link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#raft-metrics[Raft metrics]**
+
+|<prefix>.causal_clustering.core.append_index
+
+|<prefix>.causal_clustering.core.commit_index
+
+|<prefix>.causal_clustering.core.applied_index
+
+|<prefix>.causal_clustering.core.term
+
+|<prefix>.causal_clustering.core.tx_retries
+
+|<prefix>.causal_clustering.core.is_leader
+
+|<prefix>.causal_clustering.core.in_flight_cache.total_bytes
+
+|<prefix>.causal_clustering.core.in_flight_cache.max_bytes
+
+|<prefix>.causal_clustering.core.in_flight_cache.element_count
+
+|<prefix>.causal_clustering.core.in_flight_cache.max_elements
+
+|<prefix>.causal_clustering.core.in_flight_cache.hits
+
+|<prefix>.causal_clustering.core.in_flight_cache.misses
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.lag
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.bytes
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.size
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.async_put
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.sync_put
+
+|<prefix>.causal_clustering.core.message_processing_delay
+
+|<prefix>.causal_clustering.core.message_processing_timer
+
+|<prefix>.causal_clustering.core.replication_new
+
+|<prefix>.causal_clustering.core.replication_attempt
+
+|<prefix>.causal_clustering.core.replication_fail
+
+|<prefix>.causal_clustering.core.replication_maybe
+
+|<prefix>.causal_clustering.core.replication_success
+
+|<prefix>.causal_clustering.core.last_leader_message
+
+a|**link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#read-replica-metrics[Read Replica metrics] - replaced accordingly by the link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#store-copy-metrics[Store copy metrics]**
+
+|<prefix>.causal_clustering.read_replica.pull_updates
+
+|<prefix>.causal_clustering.read_replica.pull_update_highest_tx_id_requested
+
+|<prefix>.causal_clustering.read_replica.pull_update_highest_tx_id_received
+
+|===
+====
+
+=== SSL framework
+
+.See the CBC-based ciphers that are no longer used from Neo4j 2025.01:
+
+* TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+* TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_256_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_128_CBC_SHA256
+
+For more details, refer to the page on link:{neo4j-docs-base-uri}/operations-manual/5/security/ssl-framework/#ssl-other-configs[SSL framework] under the Security section.
+
+.Private keys
```

Comment:
Took it from the page, but I didn't ask anyone if it's true

Reply:
@NataliaIvakina it hasn't been changed yet, so not yet removed in the product.  Was a Q1 item that was pushed out.